app/flatpak-builtins-config.c

```diff
@@ -112,6 +112,7 @@ typedef struct
 
 ConfigKey keys[] = {
   { "languages", parse_lang, print_lang, get_lang_default },
+  { "default-languages", parse_lang, print_lang, get_lang_default },
```

Comment:
This isn't right, we should not handle the special `*` or `all` things for this setting, and there should not be a need to print a default value for it. The default of this particular key is "unset", and if you put anything in it that will normally be visible in the default output of the `languages` key. This also seems to imply that we should list the extra-languages key before the languages key.

Reply:
I see - yeah, it makes sense. However, i didn't understand how this specific piece isn't right - if I want extra-languages to be set through the cli this line should stay here, doesn't it?

Reply:
Not in its current form, because the implementations of `parse_lang()` and `print_lang()` interpret the `*`/`all` values. And instead of `get_lang_default()` for extra-languages the default is unset.